karate-core/src/main/java/com/intuit/karate/driver/DevToolsWait.java

```diff
@@ -89,14 +89,15 @@ public void setCondition(Predicate<DevToolsMessage> condition) {
         this.condition = condition;
     }
 
-    public DevToolsMessage send(DevToolsMessage dtm, Predicate<DevToolsMessage> condition) {
+    public DevToolsMessage send(DevToolsMessage dtm, Predicate<DevToolsMessage> condition, DevToolsDriver dtd) {
```

Comment:
this is great. I'll request one change, add an arg to the constructor of `DevToolsWait` so the `DevToolsDriver` can be a private final variable, so we don't need to keep passing it in the method. and in future can be used for other intelligence